src/less/hotness-signals/base/hotness-signals.less

```diff
@@ -0,0 +1,38 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.signal {
+    align-items: center;
+    border: 1px solid;
+    border-radius: 8px;
+    box-sizing: border-box;
+    display: flex;
+    font-family: MarketSans, sans-serif;
+    font-size: 10px;
+    font-weight: bold;
+    letter-spacing: 0.5px;
+    line-height: 12px;
+    padding: 1px 8px;
+
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.signal-trustworthy {
```

Comment:
Correct syntax for all these modifiers is a double-dash, e.g. `.signal--trustworthy`